src/zope/i18nmessageid/_zope_i18nmessageid_message.c

```diff
@@ -216,7 +264,7 @@ MessageType = {
         /* tp_setattro       */ (setattrofunc)0,
         /* tp_as_buffer      */ 0,
         /* tp_flags          */ Py_TPFLAGS_DEFAULT
-				| Py_TPFLAGS_BASETYPE 
+                            | Py_TPFLAGS_BASETYPE
                           	| Py_TPFLAGS_HAVE_GC,
```

Comment:
The indentation change looks wrong?

Reply:
That's because this file is full of tabs. I remove the tabs, but it makes the diff unreadable.